<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Century</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>BurnMaster</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="naming_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century</family>
      <pointsize>32</pointsize>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>BurnMaster</string>
    </property>
   </widget>
   <widget class="QLabel" name="flashing_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century</family>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Прошивка ключа</string>
    </property>
   </widget>
   <widget class="QPushButton" name="flashingButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>235</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century</family>
      <pointsize>20</pointsize>
      <fontweight>DemiBold</fontweight>
     </font>
    </property>
    <property name="text">
     <string>Прошить ключ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="keysCheckButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>250</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century</family>
      <pointsize>18</pointsize>
      <fontweight>DemiBold</fontweight>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Просмотр 
выданных ключей</string>
    </property>
   </widget>
   <widget class="QPushButton" name="orgCheckButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>370</y>
      <width>250</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century</family>
      <pointsize>18</pointsize>
      <fontweight>DemiBold</fontweight>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Просмотр 
действующих 
организаций</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>121</y>
      <width>311</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="keySelectionList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>255</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>255</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Century</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox { color: gray; } 
QComboBox QAbstractItemView { color: black; }  </string>
       </property>
       <property name="placeholderText">
        <string>Выберите ключ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="findKeyButton">
       <property name="minimumSize">
        <size>
         <width>35</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::EditFind"/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>165</y>
      <width>454</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QComboBox" name="orgSelectionList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>255</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>255</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Century</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox { color: gray; } 
QComboBox QAbstractItemView { color: black; }  </string>
       </property>
       <property name="placeholderText">
        <string>Выберите организацию</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addOrgButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Century</family>
         <pointsize>10</pointsize>
         <fontweight>DemiBold</fontweight>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">margin-left:10px;</string>
       </property>
       <property name="text">
        <string>Добавить организацию</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
